### ISSUE TITLE
"webpack: command not found" … I think some dependencies are missing.

### ISSUE THREAD

**Issue by sjlongland** [aswathkk/wboard#1]
```
RC=0 stuartl@rikishi /tmp/wboard $ yarn
yarn install v0.23.4
warning wboard@1.0.0: No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
error upath@1.0.2: The engine "node" is incompatible with this module. Expected version ">=4 <=9".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
RC=1 stuartl@rikishi /tmp/wboard $ yarn start
yarn start v0.23.4
$ npm run build && NODE_ENV=prod node server/index.js 

> wboard@1.0.0 build /tmp/wboard
> webpack --config webpack.prod.js

sh: webpack: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! wboard@1.0.0 build: `webpack --config webpack.prod.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the wboard@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/stuartl/.npm/_logs/2021-01-30T01_56_38_631Z-debug.log
error Command failed with exit code 1.
```

I think your instructions or `yarn` configuration (not familiar with this tool, I'm used to `npm`) is missing a step to install `webpack`.

**Comment by aswathkk** [comment]
Your dependency installation command `yarn` failed.
From your terminal output, I found that you are using an incompatible version of node.
```
error upath@1.0.2: The engine "node" is incompatible with this module. Expected version ">=4 <=9".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I created this project 3 years back and it is expecting the node version to be >= 4 and  <= 9
Make use of nvm to downgrade the node version and try again.

**Comment by sjlongland** [comment]
`nvm` isn't available on the platform I am using (Alpine Linux/AMD64).  So I'm stuck with whatever they have shipped, which happens to be NodeJS 12.20.1.

NodeJS 9 and earlier are no longer supported, thus no longer receive security fixes or enhancements to support newer operating systems.  NodeJS 10 will be in that boat [very soon (April)](https://nodejs.org/en/about/releases/).

If this project won't run on at least NodeJS 12, then I'm sorry to say, the project is dead… time to hit the Archive button.
